File: 26. Remove Duplicates from Sorted Array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        '''
        Two pointers
        '''
        if not nums:
            return 0
        read, write = 1, 1
        while read < len(nums):
            if nums[read] != nums[read-1]:
                nums[write] = nums[read]
                write += 1
            read += 1
        return write

File: 26. Remove Duplicates from Sorted Array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int write = 0;
        for (int j = 1; j<nums.length; j++) {
            if (nums[j] != nums[write]) {
                nums[++write] = nums[j];
            }
        }
        return write+1;
    }
}